nucleus/pom.xml

```diff
@@ -273,7 +273,7 @@
         <!-- JAX-RS -->
         <jax-rs-api.spec.version>2.1.5</jax-rs-api.spec.version>
         <jax-rs-api.impl.version>2.1.5</jax-rs-api.impl.version>
-        <jersey.version>2.29.payara-p3</jersey.version>
+        <jersey.version>2.29.payara-p4-SNAPSHOT</jersey.version>
```

Comment:
Snapshot? 
Not simply p4?

Reply:
You're right, it should be waiting yet for a review of the jersey and after that I shoud run into this PR with the final version.